src/patterns/ec2-app.ts

```diff
@@ -87,11 +87,11 @@ export class GuEc2App {
       port: props.applicationPort,
     });
 
-    new GuApplicationListener(scope, "Listener", {
+    new GuHttpsApplicationListener(scope, "Listener", {
```

Comment:
Should we have tests that cover this change?

Reply:
This is covered by the [snapshot test for this pattern](https://github.com/guardian/cdk/blob/314af33330c2445c4d8de779dbedd9ea6507bffe/src/patterns/ec2-app.test.ts#L9-L30). The resulting CFN is actually identical, the pattern code is just slightly simpler to read (IMO).